sdk/core/core-auth/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.3.1 (Unreleased)
 
+- Added `tenantId`, which is used to enable multi-tenant authentication.
```

Comment:
nit: perhaps we should mention the interface this was added to?

Reply:
Ohh true! Thank you!!!